platform/platform-configuration-listener/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -22,6 +22,7 @@
             <ext:property name="configFileDirectory" value="${ddf.home}/etc"/>
             <ext:property name="processedDirectory" value="${ddf.home}/etc/processed"/>
             <ext:property name="failedDirectory" value="${ddf.home}/etc/failed"/>
+            <ext:property name="exportedDirectory" value="${ddf.home}/etc/exported"/>
```

Comment:
It's not roundabout based on how we're doing the configurations at the moment (none of the directories are configurable), and this isn't expected to be used by a CLI necessarily (we just happen to hook into it).  If we get direct feedback from Users / Admins on what they're trying to accomplish and it suggests they need this capability, then we can easily build it in then.  

For the time being I'd rather keep it scoped to the original ticket.